examples/amplitude_analysis/sketch.js

```diff
@@ -4,13 +4,15 @@
 
 var soundFile;
 var amplitude;
+var bufferSize = 2048; // a value which can be multiple of 2 , eg. 256 , 512 , 1024 , 2048 ..etc
```

Comment:
looks like this comment should be updated to read "power of 2"

Reply:
ya ofcourse !!